src/sentry/tasks/store.py

```diff
@@ -307,8 +310,9 @@ def _do_symbolicate_event(cache_key, start_time, event_id, symbolicate_task, dat
 @instrumented_task(
     name="sentry.tasks.store.symbolicate_event",
     queue="events.symbolicate_event",
-    time_limit=65,
-    soft_time_limit=60,
+    time_limit=settings.SYMBOLICATOR_PROCESS_EVENT_HARD_TIMEOUT + 30,
+    soft_time_limit=settings.SYMBOLICATOR_PROCESS_EVENT_HARD_TIMEOUT + 20,
+    acks_late=True,
```

Comment:
`acks_late=True` means the task will be acked only after completion and tasks killed during deploy will be re-enqueued.